src/stories/common.tsx

```diff
@@ -148,12 +148,23 @@ export const CommonCustomStoryWithoutLabelProps = (referenceKey: string): ArgTyp
 export const COLUMNS_ARG_TYPE: ArgTypes = {
 	columns: {
 		description: `Specifies the number of columns to be span across in desktop / tablet / mobile viewports. If an array is specified, the format is as such <code>[startCol, endCol]</code>.<br><br>
```

Comment:
explain how the grid system between v2 and v3 are applied automatically by detecting v2 / v3 breakpoints